thirdparty/runtime_environments.go

```diff
@@ -0,0 +1,61 @@
+package thirdparty
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/evergreen-ci/gimlet"
+	"github.com/pkg/errors"
+)
+
+type RuntimeEnvironmentsClient struct {
+	Client  *http.Client
+	BaseURL string
+	APIKey  string
+}
+
+func NewRuntimeEnvironmentsClient(baseURL string, apiKey string) *RuntimeEnvironmentsClient {
+	c := RuntimeEnvironmentsClient{
+		Client:  &http.Client{},
+		BaseURL: baseURL,
+		APIKey:  apiKey,
+	}
+	return &c
+}
+
+// getImageNames returns a list of strings containing the names of all images from the runtime environments api
```

Comment:
From our [Go Coding Conventions](https://docs.google.com/document/d/1-DB1FQ_zb5fX_0pnVcwyU0HmJPrloFBu6qSE8dkk8PU/edit#heading=h.do73fevfgf93), comments should be full sentences with proper punctuation! Also would recommend capitalizing the word API since it is an acronym.